airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -147,7 +147,6 @@ class KubernetesPodOperator(BaseOperator):
         to populate the environment variables with. The contents of the target
         ConfigMap's Data field will represent the key-value pairs as environment variables.
         Extends env_from.
-    :param: kubernetes_conn_id: To retrieve credentials for your k8s cluster from an Airflow connection
```

Comment:
This docstring is duplicated; there’s already one near the top, so I took the chance to remove it.